loleaflet/src/control/Control.UIManager.js

```diff
@@ -84,6 +84,10 @@ L.Control.UIManager = L.Control.extend({
 				jsontype: 'dialog',
 				type: 'modalpopup',
 				children: [
+					{
+						id: 'jsdialog-overlay',
+						type: 'container'
```

Comment:
it shouldn't be done here, in the future I plan to reuse it, it would be not good if you need to remember that you need to add this overlay

Reply:
could you create new control in JSDialogBuilder which creates div with CLASS jsdialog-overlay?

Reply:
understood, but isn't this type `modalpopup` supposed to be for those situations where there is really just one dialog and its overlay?

Reply:
ok